pandas/tests/indexes/conftest.py

```diff
@@ -50,3 +51,51 @@ def zero(request):
     # For testing division by (or of) zero for Index with length 5, this
     # gives several scalar-zeros and length-5 vector-zeros
     return request.param
+
+
+def _create_series(index):
```

Comment:
can we do this as another step. I am really not in love with the way these fixtures are created, they are duplicative of many other fixtures and named in a funny way.

Reply:
I completely agree that these fixtures feel weird. However, I need them to remove the `Ops` Mixin, which is basically the goal of this PR.

...and even though they are weird, the same weirdness is going on already in the code. So it's at least not making anything worse ✌️